fix(sdk build, package.json): Added .proto file as a part of build files and used it in grpc connection.

Hermione2408 opened the pull request:
Previously we were using cp command to copy services.proto file from proto folder to dist folder but cp command doesnt work for windows and hence the build fails in windows. I added "copy-files" script which uses copyfiles npm module to copy proto file and is OS independent.

fix keploy/keploy#372

Hermione2408 committed:
fix(sdk build, package.json): added a copy-files script which is OS independent

Previously we were using cp command to copy services.proto file from proto folder to dist folder but
cp command doesnt work for windows and hence the build fails in windows. I added "copy-files" script
which uses copyfiles npm module to copy proto file and is OS independent.

fix #51

Signed-off-by: Hermione Dadheech <hermionedadheech@gmail.com>

re-Tick commented:
Hi @Hermione2408, Please git rebase to pass DCO. You can follow the 3 steps mentioned in the failing DCO job

re-Tick commented:
I think that we should include .proto file in our "files" of package.json to include them in the npm build files.
<img width="633" alt="Screenshot 2023-03-01 at 10 19 10 PM" src="https://user-images.githubusercontent.com/60597329/222206888-c9a19380-1315-41ba-bbb1-f400f77d87ab.png">

And use relative path to that for grpcClient connection.

Hermione2408 committed:
fix(package.json, mock.ts, keploy.ts, grpc.proto): added proto file in build for grpc connection

Added .proto file as a part of build file and used it in grpc connection.

fix #51

Signed-off-by: Hermione Dadheech <hermionedadheech@gmail.com>

Hermione2408 commented:
@re-Tick I have updated the PR with the changes Please review it and let me know if this need any changes.

re-Tick commented:
Hi @Hermione2408, Can we use the same "services.proto" file from photo directory instead of new "graph.proto".

Hermione2408 committed:
fix: adding serives.proto instead of new grpc.proto

fix #51

Signed-off-by: Hermione Dadheech <hermionedadheech@gmail.com>

Hermione2408 commented:
Hey @re-Tick do you mean something like this?

re-Tick commented:
Yes @Hermione2408. Have you built and tested your changes locally?

Hermione2408 commented:
Yes I have built and tested my changes locally ,Its working fine.
